File: LeetCode/C++/1. Two Sum.cpp
#include<stdio.h>
#include<malloc.h>
int* twoSum(int* nums, int numsSize, int target) 
{
	int i,j;
	int* result=(int*)malloc(sizeof(int)*2);
    for(i=0;i<numsSize-1;i++)
    {
    	j=i+1;
    	while(j<numsSize)
    	{
    		if(nums[i]+nums[j]==target)
    		{
    			result[0]=i;
    			result[1]=j;
    			return result;
			}
    		j++;
		}
	}
    return result;
}

File: LeetCode/JavaScript/1. Two Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
    var map = new Map();
    nums.forEach(function (value, index) {
        map.set(value, index);
    });
    for (let i = 0; i < nums.length; i++) {
        let find = target - nums[i];
        if (map.has(find) && map.get(find) !== i) {
            return [i, map.get(find)];
        }
    }
};